js/jquery.socialfeed.js

```diff
@@ -732,6 +746,8 @@ if (typeof Object.create !== 'function') {
 
                     getPosts: function(json) {
                         console.log(json);
```

Comment:
there is a consiole.log to remove plus i think is better to avoid any calculation if there are no feed to render anyway:

`                    getPosts: function(json) {
                        if (json.query.count > 0 ){
							// fix rendering if posts_to_load_count is not reached
							if (json.query.count < options.rss.limit) posts_to_load_count -= options.rss.limit - json.query.count;
                            $.each(json.query.results.feed, function(index, element) {
                                var post = new SocialFeedPost('rss', Feed.rss.utility.unifyPostData(index, element));
                                post.render();
                            });
                        }
                    },`